specification/paths/Collections-collection_id.json

```
@@ -26,6 +26,38 @@
         "schema": {
           "type": "string"
         }
+      },
+      {
+        "name": "filter[collection_date]",
+        "in": "query",
+        "description": "Date string in ISO 8601 date format (YYYY-MM-DD). Only collections with a collection date that matches this date will be in the response.",
+        "schema": {
+          "type": "string"
+        }
+      },
+      {
+        "name": "filter[shop]",
+        "in": "query",
+        "description": "Comma separated string of shop ids to filter by.",
+        "schema": {
+          "type": "string"
+        }
```

```suggestion
        "$ref": "#/components/parameters/query-filter-shop"
```
FYI - there is already a `query-shipments-filter-shop` which we should rename (and also the references to it).